syntax = "proto3";
package tinkoff.proto;

// Request
message EditBeneficiaryBankDetailsRequest {
  string beneficiary_id = 1;
  string bank_details_id = 2;
  string data = 3;
}
